RecoHI/HiTracking/python/HILowPtConformalPixelTracks_cfi.py

```diff
@@ -130,6 +130,72 @@
     hiConformalPixelTracks
 )
 
+from Configuration.ProcessModifiers.gpu_cff import gpu
+from Configuration.ProcessModifiers.pixelNtupletFit_cff import pixelNtupletFit
+from RecoTracker.PixelTrackFitting.pixelTrackSoAFromCUDAHIonPhase1_cfi import pixelTrackSoAFromCUDAHIonPhase1 as _pixelTracksSoA
+from RecoTracker.PixelSeeding.caHitNtupletCUDAHIonPhase1_cfi import caHitNtupletCUDAHIonPhase1 as _pixelTracksCUDA
+from RecoTracker.PixelTrackFitting.pixelTrackProducerFromSoAHIonPhase1_cfi import pixelTrackProducerFromSoAHIonPhase1 as _pixelTrackProducerFromSoA
+
+from HeterogeneousCore.CUDACore.SwitchProducerCUDA import SwitchProducerCUDA
+
+hiPixelTracksCUDA = _pixelTracksCUDA.clone(pixelRecHitSrc="siPixelRecHitsPreSplittingCUDA", idealConditions = False,
+        ptmin = 0.25, hardCurvCut = 0.0756, doPtCut = False,
+        onGPU = True,
+        phiCuts = cms.vint32([900 for i in range(19)]), #19 pairs
+        trackQualityCuts = cms.PSet(
```

Comment:
shouldn't this be 
```suggestion
        trackQualityCuts = dict(
```
and then drop all `cms.sometype` specifications